src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -293,8 +284,8 @@ void GEO_CortexPrimitive::copyPrimitive( const GEO_Primitive *src, GEO_Point **p
 	m_object = orig->m_object;
 	
 	GA_VertexWrangler vertexWrangler( *getParent(),	*orig->getParent() );
-	
-	GA_Offset v = m_offset;
```

Comment:
I have no idea, which is why I'd be afraid to change it, especially since we know it's going away entirely in the future.